sunpy/time/timerange.py

```diff
@@ -405,7 +405,8 @@ def __contains__(self, time):
         """
         Checks whether the given time lies within this range. Both limits are
```

Comment:
We should modify this docstring to be more logical for `.contains()` and also to demonstrate the `t in tr` syntax as well.